.github/workflows/interface-unit-tests.yml

```
@@ -219,7 +257,9 @@ jobs:
       - setup-ci-load
       - determine_runner
       - default-dependency-versions
+      - disable-fail-fast-wae
     strategy:
+      fail-fast: ${{ contains(fromJSON(needs.disable-fail-fast-wae.outputs.fail_fast),  'default') }}
```

```suggestion
      fail-fast: ${{ needs.disable-fail-fast-wae.outputs.fail_fast == 'default' }}
```
Since the output of the job is a simple string now, we can further simplify this logic as well (goes for all the jobs)